server/src/main/java/org/apache/uniffle/server/storage/SingleStorageManager.java

```diff
@@ -149,4 +164,15 @@ public void start() {
   public void stop() {
     // do nothing
   }
+
+  public void removeAppStorageInfo(PurgeEvent event) {
+    String appId = event.getAppId();
+    ApplicationStorageInfo info = appStorageInfoMap.remove(appId);
+    if (info != null) {
+      ShuffleServerMetrics.gaugeStorageUsedBytes.dec(info.getUsedBytes());
+      ShuffleServerMetrics.gaugeFlushFileNum.dec(info.getFileNum());
```

Comment:
We do not need to dec these two metrics, and we can call these `counterStorageWriteBytes` and `counsterFlushFileNum`, base on this, we can get currentStorageUsedBytes by `counterStorageWriteBytes-counterTotalDeleteDataSize `